packs/paizo-pregens/bottlespeaker-level-4.json

```diff
@@ -4069,7 +4069,7 @@
         "saves": {},
         "skills": {
             "acr": {
-                "rank": 0
+                "rank": 2
```

Comment:
```suggestion
                "rank": 0
```

He's not trained in acrobatics.

![image](https://github.com/foundryvtt/pf2e/assets/80183198/1d539cd3-d6af-41d4-ac2a-6181cbc05ffd)